Added black-duck-panel.yaml

idealphase opened the pull request:
### Template / PR Information

<!-- Explains the information and/or motivation for update or/ creating this templates -->
<!-- Please include any reference to your template if available -->

- Added black-duck-panel.yaml

### Template Validation

I've validated this template locally?
- [X] YES
- [ ] NO


#### Additional Details (leave it blank if not applicable)
![black-duck-panel-UI1](https://user-images.githubusercontent.com/12832679/175067048-9cd2f1c8-2668-4728-a159-cc34f22f43c6.png)
![black-duck-panel-UI2](https://user-images.githubusercontent.com/12832679/175067059-6d5d6a9c-b2c1-4846-93e5-989d44f4e343.png)
![black-duck-panel-shodan](https://user-images.githubusercontent.com/12832679/175067075-b884fd78-cb54-4923-9b82-5c3d7f062b5e.png)
![black-duck-panel-nuclei](https://user-images.githubusercontent.com/12832679/175067090-68c218ea-b91e-47f0-b8c5-0912c31b8949.png)

### Additional References:

- [Nuclei Template Creation Guideline](https://nuclei.projectdiscovery.io/templating-guide/)
- [Nuclei Template Matcher Guideline](https://github.com/projectdiscovery/nuclei-templates/wiki/Unique-Template-Matchers)
- [Nuclei Template Contribution Guideline](https://github.com/projectdiscovery/nuclei-templates/blob/master/CONTRIBUTING.md)
- [PD-Community Discord server](https://discord.gg/projectdiscovery)

ritikchaddha commented:
Hello @idealphase, I suggest adding this matcher to avoid getting FP. 
```
      - type: word
        part: body
        words:
          - 'ProtexLoginPage'
          - 'CCLoginPage'
        condition: or
```

idealphase commented:
Thank you so much @ritikchaddha for your suggestion. I updated additional matcher as your suggestion. and also added your name in author part. :) Thanks bro.

princechaddha commented:
Thankyou @idealphase @ritikchaddha
